controllers/update_status.go

```diff
@@ -133,6 +137,7 @@ func (updateStatus) reconcile(ctx context.Context, r *FoundationDBClusterReconci
 		for _, coordinator := range databaseStatus.Client.Coordinators.Coordinators {
 			address, err := fdbv1beta2.ParseProcessAddress(coordinator.Address.String())
 			if err != nil {
+				log.Info("update_status skipped due to error in ParseProcessAddress", "Error", err)
```

Comment:
all such log message in this file are changed to error message and assigned to curError.